docs/intermittent_producer.md

```diff
@@ -0,0 +1,12 @@
+### Intermittent Producer Handling
+Since v3.1.0 of the Producer C++ SDK by default we have automatic handling for intermittent streaming.  This comes at a cost of using 1 extra thread to manage the timer used by the SDK to automatically signal to the KVS back-end that a fragment can be closed out.  If on a prior release of the SDK you were manually setting fields like `stream_info_.streamCaps.frameOrderingMode` you no longer should do that.  An example of this scenario is if you stream for an amount of time, then you stop producing frames (for example the case of a security camera which is only uploading video/audio during a motion event) and then after an unknown amount of time (for example the next motion event) you start streaming again.  This is all automatically handled by the SDK now.
```

Comment:
Recommend starting the paragraph by defining what we mean by intermittent streaming, then telling how clients and SDK work in this scenario today, and lastly telling what exactly is "automatically" handled in v3.1. Current flow is roughly backwards, which makes it difficult to create a mental picture of what is happening

Reply:
Makes sense, see new commits.